fuzz/build.sh

```diff
@@ -54,7 +54,7 @@ fi
 
 
 cmake -B $BUILD -S . \
-      -DSIMDUTF_TESTS=Off \
+      -DSIMDUTF_TESTS=On \
```

Comment:
please instead use a variable ENABLE_TESTS which is set to On in the else branch on row 46 and Off otherwise.

Reply:
Thanks for the suggestion. I have fixed accordingly.